test-src/base.spec.js

```diff
@@ -106,14 +106,14 @@ describe('BaseManager', function() {
             });
 
             // issue 51
-            it('should use the same version of webgme as the setup tool', function() {
+            it('should use the same major version of webgme as the setup tool has for webgme-engine', function() {
                 // Compare the package.json values
                 var packageJSON = path.join(initProject, 'package.json'),
                     toolJson = path.join(__dirname, '..', 'package.json'),
                     toolDeps = require(toolJson).dependencies,
                     deps = require(packageJSON).peerDependencies;
 
-                assert.equal(deps.webgme, toolDeps.webgme);
+                assert.equal(deps.webgme, toolDeps['webgme-engine']);
```

Comment:
This isn't currently checking the major version... I think this test will fail after a patch/minor release of webgme (without an update to webgme-engine), right?

Reply:
👍

Reply:
Actually no. It's not reading the installed version but rather the required version which is ^2.0.0.